core/src/graph/mod.rs

```diff
@@ -744,7 +744,11 @@ pub mod tests {
                             "generator": "safe_email"
                         }
                     },
-                    "num_logins_again": "@users.content.num_logins"
+                    "num_logins_again": "@users.content.num_logins",
+                    "a_large_number": {
+                        "type": "number",
+                        "range": {}
```

Comment:
what happens if the user completely omits the `"range"` entry?

Reply:
It is interpreted as `"range": {}`, see the [notes on the PR](https://github.com/getsynth/synth/pull/146#issue-996021289).